presto-kudu/src/test/java/io/prestosql/plugin/kudu/KuduServer.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kudu;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.net.HostAndPort;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+
+import java.io.Closeable;
+import java.util.List;
+
+public class KuduServer
+        implements Closeable
+{
+    private static final Integer KUDU_MASTER_PORT = 7051;
+    private static final Integer KUDU_TSERVER_PORT = 7050;
+
+    private final GenericContainer<?> masterContainer;
+    private final List<GenericContainer<?>> tServerContainers;
+
+    public KuduServer(int numberOfReplica)
```

Comment:
maybe use always 2, no need to pass parameter then.

Reply:
We can't set an even number of replication